webcompat/static/js/lib/bugform.js

```diff
@@ -491,8 +499,13 @@ function BugForm() {
       img_url,
       ")"
     ].join("");
+    var delimeter = "[![Screenshot Description](";
     this.descField.val(function(idx, value) {
-      return value + "\n\n" + imageURL;
+      if (value.includes(delimeter)) {
```

Comment:
(yeah, was just going through all my github notifications... not really reviewing. 😄 )